app/homepage.py

```diff
@@ -77,70 +78,54 @@ def meteo_graphs(display=False):
 # Content and App layout
+# The following function is used in the main.py file to instantiate the layout of the homepage
 
-# Instantiating the alerts map object from alerts.py and setting it as the default map object:
-map_object = build_alerts_map()
-
-# Instantiating fig objects from graphs.py functions
-meteo_fig = generate_meteo_fig()
-
-# Instantiating navbar object from navbar.py
-nav = Navbar()
-
-# Instantiating map layers button object from alerts.py
-map_layers_button = build_layer_style_button()
-
-# Instantiating map style button object
-map_style_button = build_map_style_button()
-
-# Instantiating meteo graphs, set to True to display them under the map, False to hide them
-meteo_graphs = meteo_graphs(display=False)
-
-# Body container
-body = dbc.Container([
-    dbc.Row(
-        [dbc.Col(html.H1('Plateforme de Monitoring', style={'text-align': 'center'}), className="pt-4"),
-         ]),
-    dbc.Row(
-        [
-            dbc.Col(
-                dcc.Dropdown(
-                    id='user_department_input',
-                    options=[
-                        {'label': 'Ardèche', 'value': 'Ardèche'},
-                        {'label': 'Gard', 'value': 'Gard'},
-                        {'label': 'Landes', 'value': 'Landes'}],
-                    placeholder="Départements"),
-                md=3),
-        ]
-    ),
-    dbc.Row(
-        [dbc.Col(
-            [
-                dcc.Markdown('---'),
-                # map filters added here
-                html.H5(("Filtres Carte"), style={'text-align': 'center'}),
-                html.P(map_layers_button),
-                dcc.Markdown('---'),
-                html.P(map_style_button),
-                html.P(id="hp_slider"),
-            ],
-            md=3),
-         dbc.Col(
-            # map object added here
-            html.Div(map_object, id='hp_map'),
-            md=9)]
-    ),
-    # meteo graphs added here
-    meteo_graphs
-],
-    fluid=True,
-)
-
-
-# Gathering all these elements in a HTML Div and having it returned by the Homepage function
 def Homepage():
 
-    layout = html.Div([nav, body])
+    # Body container
+    body = dbc.Container([
+        dbc.Row(
+            [dbc.Col(html.H1('Plateforme de Monitoring', style={'text-align': 'center'}), className="pt-4"),
+             ]),
+        dbc.Row(
+            [
+                dbc.Col(
+                    dcc.Dropdown(
+                        id='user_department_input',
+                        options=[
+                            {'label': 'Ardèche', 'value': 'Ardèche'},
+                            {'label': 'Gard', 'value': 'Gard'},
+                            {'label': 'Landes', 'value': 'Landes'}],
+                        placeholder="Départements"),
+                    md=3),
+            ]
+        ),
+        dbc.Row(
+            [dbc.Col(
+                [
+                    dcc.Markdown('---'),
```

Comment:
This part of the code has been changed too in my PR. I guess the first PR that will be merged won't have to deal with conflicts 😅